dev-docs/plugins/index.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: page
+title: Prebid Plugins
+description: Prebid Plugins
+top_nav_section: dev_docs
+nav_section: reference
+pid: 28
+---
+
+<div class="bs-docs-section" markdown="1">
+
+# Prebid Plugins
+
+Prebid plugins allow you to extend your header bidding solutions.
```

Comment:
Please describe the general concept of plugins more. I was initially confused about the difference between a plugin and a module. After reading it, I wonder if there will ever be any more plugins -- maybe this is a Brightcove-specific thing?

Maybe something like "A 'plugin' is Prebid presence in another technology platform."